### ISSUE TITLE
Need to be able everywhere within REACH to expand or reduce MAP (to any size) - function responsive

### ISSUE THREAD

**Issue by LucieGueuning** [MSFREACH/msf-reach#274]

**Comment by matthewberryman** [comment]
@mehrdadgit please do for all maps bar the reports and contact cards (but do do it for reports modal). https://jqueryui.com/resizable/

**Comment by mehrdadgit** [comment]
Is this still a priority/necessary after #270 has been done?

**Comment by matthewberryman** [comment]
What with other changes to the way an event is defined and mapped, plus #363 (dropping modal anyway) let’s keep this as low priority until I’ve had a chance to revisit and also talk through with @anexact

Sent from my iPhone

On 19 Apr 2018, at 21:02, Mehrdad <notifications@github.com<mailto:notifications@github.com>> wrote:


Is this still a priority/necessary after #270<https://github.com/MSFREACH/msf-reach/issues/270> has been done?

**Comment by matthewberryman** [comment]
@mehrdadgit please work on this after your contact edit/deletion work
